AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Controllers/SiteAdminControllerShould.cs

```diff
@@ -31,11 +34,15 @@ public void IndexReturnsCorrectViewModel()
             await TaskFromResultZero;
         }
 
-        [Fact(Skip = "NotImplemented")]
-        public async Task DeleteUserHasHttpGetAttribute()
+        [Fact]
+        public void DeleteUserHasHttpGetAttribute()
         {
-            //delete this line when starting work on this unit test
-            await TaskFromResultZero;
+            var mediator = new Mock<IMediator>();
+            var controller = new SiteController(null, null, mediator.Object);
```

Comment:
you don't need to create a mediator mock, because the method you're testing doesn't invoke the mediator. To create the controller, pass `null` instead of `mediator.Object`